src/components/App.tsx

```diff
@@ -301,6 +305,11 @@ class App extends React.Component<AppProps, AppState> {
       }
       readyPromise.resolve(api);
     }
+
+    this.excalidrawContainerValue = {
+      container: this.excalidrawContainerRef.current,
```

Comment:
haven't checked: isn't this `null` when `constructor` runs?

Reply:
yeah my bad,  good catch 👍🏻